### ISSUE TITLE
add option to resign trophy folders in bulk

### ISSUE THREAD

**Issue by Downgrade1998** [bucanero/apollo-ps3#78]

**Comment by bucanero** [comment]
the trophy resign was meant for those who wanted to edit/unlock trophies, and that's why it wasn't added as a bulk option.

I guess the only case for a bulk resign of trophies would be if you actually overwrite or copy trophy-sets manually. Could you confirm why you need to resign trophies in bulk?

**Comment by rocknard** [comment]
In my case would be because when you install a new hdd or format it, you will want to restore all trophies at the same time (i used to backup them with multiman and the saves too, because some of them have "anti copy" protection).

P.S.: sorry the hijack.

**Comment by rocknard** [comment]
Sorry, i cant edit my comments (win xp and navigator handicap):

I don't know how reliable is the official "rebuild database" when i tried to restore all my trophies, because some if not all wasn't actually "installed". Maybe i need to open the game firstly and then do the "rebuild database" with my saved trophies (a hastle if you hay so many games).

**Comment by Don-Keballs** [comment]
I agree this feature would be helpful. I've recently moved from one PS3 to another and the feature to bulk resign would be very useful. 

That said, It's not a major issue. I don't play online or plan to sync my trophies. But I do find them nice to look at and add to when I come back to a game.

If it's not too much of a headache for you it would be nice to add this feature.

P.s. Would it be possible to also bulk import trophies from a USB? There's an option to export in bulk but not import.
